src/api/topics/routes.py

```diff
@@ -20,16 +20,20 @@
 
 
 @ns.route("")
-class TopicList(Resource):  # type: ignore
+class Topics(Resource):  # type: ignore
```

Comment:
Why did we change this? Seems like `TopicList` is more consistent with our `PostsList` and `UserList`. And if we really like to have `Topics` instead of `TopicList`, we need to change `UserList` and `PostsList` as well. And it should be in a separate ticket.